packages/fluentui/react-northstar/src/components/Popup/Popup.tsx

```diff
@@ -505,6 +506,8 @@ export const Popup: React.FC<PopupProps> &
     if (open) {
       // when new state 'open' === 'true', save the last focused element
       updateTriggerFocusableRef();
+
+      setVirtualParent(popupContentRef.current, triggerRef.current);
```

Comment:
I wish I could add this to `PortalInner` somehow, but that component never automatically created an inner `div` as `Layer` does, so we can't guarantee existence of an element to mark.

Reply:
Is this call valid? 😕  In `packages/office-ui-fabric-react/src/components/Layer/Layer.base.tsx` it will be called with `setVirtualParent(layerElement, this._rootRef.current!);` where:
- `layerElement` is an element where portal's content will be rendered
- some `span` element that will be always rendered inline 

In our case `triggerRef` points to some element, but it's optional and can be missing as `trigger` is not required...

Reply:
It was extremely difficult looking at this code and even debugging to figure out which element was the popup element and which was the host. Some guidance would be appreciated.